tests/page-objects/forms/z-score.po.js

```diff
@@ -133,65 +134,64 @@ const docs = [
-    browser.wait(() => {
-      return addButton.isPresent();
-    }, 10000);
-    helper.clickElement(addButton);
-    element(by.css('.action-container .general-actions .dropup.open .dropdown-menu li:first-child a')).click();
+    await helper.waitUntilReadyNative(addButton);
+    await helper.clickElement(addButton);
+    const form = element(by.css('.action-container .general-actions .dropup.open .dropdown-menu li:last-child a'));
+    await helper.clickElement(form);
 
-    browser.wait(() => {
+    await browser.wait(() => {
       return element(by.css('[name="/data/my_sex"][value="female"]')).isPresent();
     }, 10000);
   },
 
-  submit: () => {
-    const submitButton = element(by.css('[ng-click="onSubmit()"]'));
-    helper.waitElementToBeClickable(submitButton);
-    submitButton.click();
-    helper.waitElementToBeVisible(element(by.css('div#reports-content')));
+  submit: async () => {
+    const submitButton = element(by.css('.btn.submit.btn-primary'));
+    await helper.waitElementToBeClickable(submitButton);
+    await submitButton.click();
+    await helper.waitElementToBeVisible(element(by.css('div#reports-content')));
   },
 
   reset: () => {
     element(by.css('.icon.icon-refresh')).click();
   },
 
-  setHeight: height => clearAndFill(element(by.css(`[name="/data/my_height"]`)), height),
-  setWeight: weight => clearAndFill(element(by.css(`[name="/data/my_weight"]`)), weight),
-  setAge: age => clearAndFill(element(by.css(`[name="/data/my_age"]`)), age),
-  setSex: sex => element(by.css(`[name="/data/my_sex"][value="${sex}"]`)).click(),
+  setHeight: async height => await clearAndFill(element(by.css(`[name="/data/my_height"]`)), height),
+  setWeight: async weight => await clearAndFill(element(by.css(`[name="/data/my_weight"]`)), weight),
+  setAge: async age => await clearAndFill(element(by.css(`[name="/data/my_age"]`)), age),
+  setSex: async sex => await element(by.css(`[name="/data/my_sex"][value="${sex}"]`)).click(),
 
-  setPatient: patient => {
-    module.exports.setSex(patient.sex);
-    module.exports.setAge(patient.age);
-    module.exports.setHeight(patient.height);
-    module.exports.setWeight(patient.weight);
+  setPatient: async patient => {
+    await module.exports.setSex(patient.sex);
+    await module.exports.setAge(patient.age);
+    await module.exports.setHeight(patient.height);
+    await module.exports.setWeight(patient.weight);
   },
 
-  getHeightForAge: () => clickAndGetValue(element(by.css('[name="/data/hfa"]'))),
-  getWeightForAge: () => clickAndGetValue(element(by.css('[name="/data/wfa"]'))),
-  getWeightForHeight: () => clickAndGetValue(element(by.css('[name="/data/wfh"]'))),
+  getHeightForAge: async () => await clickAndGetValue(element(by.css('[name="/data/hfa"]'))),
+  getWeightForAge: async () => await clickAndGetValue(element(by.css('[name="/data/wfa"]'))),
+  getWeightForHeight: async () => await clickAndGetValue(element(by.css('[name="/data/wfh"]'))),
```

Comment:
Similar to the note above, are the `await`s necessary on these (vs just letting them happily return promises)?